app/Resources/views/page.html.twig

```diff
@@ -256,77 +256,43 @@
 
     </div>
 
-    {% block javascripts deferred %}
-    {% for javascript in javascripts %}
-        <script async src="{{ asset(javascript) }}"></script>
-    {% endfor %}
-
     {% if hypothesis ?? false %}
```

Comment:
Might as well make it wrap `window.elifeConfig.hypothesis` instead now. Means non-Hypothesis pages create an empty object, but means we don't have to change it later.

Reply:
Good spot.